lib/tlsca/ca.go

```diff
@@ -399,6 +406,14 @@ var (
 	// allowed Azure identity into a certificate.
 	AzureIdentityASN1ExtensionOID = asn1.ObjectIdentifier{1, 3, 9999, 1, 17}
 
+	// AppGCPServiceAccountASN1ExtensionOID is an extension ID used when encoding/decoding
+	// the chosen GCP service account into a certificate.
+	AppGCPServiceAccountASN1ExtensionOID = asn1.ObjectIdentifier{1, 3, 9999, 1, 18}
+
+	// GCPServiceAccountASN1ExtensionOID is an extension ID used when encoding/decoding
+	// the list of allowed GCP service accounts into a certificate.
+	GCPServiceAccountASN1ExtensionOID = asn1.ObjectIdentifier{1, 3, 9999, 1, 19}
```

Comment:
```suggestion
	GCPServiceAccountsASN1ExtensionOID = asn1.ObjectIdentifier{1, 3, 9999, 1, 19}
```
nit: change to plural form?

Reply:
Fine with me.